web/components/PetPostModal/PetPostModal.tsx

```diff
@@ -3,18 +3,55 @@ import "react-responsive-carousel/lib/styles/carousel.min.css";
 import {
   Button,
   Flex,
+  ListItem,
   Modal,
   ModalContent,
   Stack,
   Text,
+  UnorderedList,
 } from "@chakra-ui/react";
 import ImageSlider from "./ImageSlider";
 import PetPostTagGroup from "./PetPostTagGroup";
+import PetPostListGroup from "./PetPostListGroup";
+import {
+  breedLabels,
+  genderLabels,
+  sizeLabels,
+  ageLabels,
+  IPost,
+  behavioralLabels,
+  medicalLabels,
+  spayNeuterStatusLabels,
+  houseTrainedLabels,
+  crateTrainedLabels,
+  GoodWith,
+  Trained,
+  goodWithLabels,
+} from "../../utils/types/post";
 
 const PetPostModal: React.FC<{
   isOpen: boolean;
   onClose: () => void;
-}> = ({ isOpen, onClose }) => {
+  postData: IPost;
+}> = ({ isOpen, onClose, postData }) => {
+  const behavioralTagLabels: String[] = [];
+  postData.behavioral?.map((tag) => {
+    behavioralTagLabels.push(behavioralLabels[tag]);
+  });
```

Comment:
`.map()` returns a new array so you don't have to push.

Try something like this:
```js
const behavioralTagLabels = postData.behavioral?.map((tag) => behavioralLabels[tag])
```